apps/src/applab/appStorage.js

```diff
@@ -38,6 +38,10 @@ var handleGetKeyValue = function(onSuccess, onError) {
   if (this.readyState !== 4) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest

We could create constants in our code for UNSENT, OPENED, HEADERS_RECEIVED, LOADING, and DONE to make it more readable.